playwright-e2e/tests/smoke-test.spec.ts

```diff
@@ -134,4 +134,7 @@ test("Smoke Test @smoke-test @accessibility", async ({
   });
 
 expect(accessibilityScanResults.violations).toEqual([]);
+    // Automatically uses the shared AxeBuilder configuration,
+  // but supports additional test-specific configuration too
+  .analyze();
 });
```

Comment:
hi charles, this needs to be changed to match the master as i think it was removed by mistake: 

const accessibilityScanResults = await makeAxeBuilder()
  // Automatically uses the shared AxeBuilder configuration,
  // but supports additional test-specific configuration too
  .analyze();

  await testInfo.attach('accessibility-scan-results', {
    body: JSON.stringify(accessibilityScanResults, null, 2),
    contentType: 'application/json'
  });

expect(accessibilityScanResults.violations).toEqual([]);
});